docs/development/pigs/pig-013.rst

```diff
@@ -0,0 +1,451 @@
+popular macOS and Linux package manager), or to provide Docker images.
+
+For Linux distributions like e.g. Debian, we plan to give installation
+instructions where Python and some dependencies are system-installed, and then
+on top of that the rest is pip-installed. This will be a way for advanced users
+to install Gammapy, and depends on how well the binary wheels work, for now the
+one recommended way to install it on any platform will be conda.
+
+Outlook
+=======
+
+This PIG describes only the status and very short term plan for Gammapy v0.14
+and v1.0! We expect that things will evolve in the coming years to follow the
+advancements in Python scientific computing, the scope of Gammapy and it's user
+community, and the skills and manpower of the Gammapy developer team. E.g. we
+might start using ``numba`` or ``dask`` or ``ray`` in the near future, or if
+Tensorflow continues to mature and improve in the coming years (see ``jax`` and
+``tensorflow-probability``), that could be considered for Gammapy.
+
+It is not described here, but very likely after implementing this PIG we will
+follow up by improving Gammapy testing (`GH 2270`_) and modernise the Gammapy
+packaging following the recommendations in `APE 17`_, and we might also change
+the Gammapy git and release workflow to have stable branches with backports
+after v1.0.
+
+Alternatives
+============
+
+Since we have a working setup already, we could do nothing, which seems nice at
+first, but really means continued time sink and suffering by the Gammapy
+maintainers of the continuous integration testing system and the release
+manager.
+
+The parts here that require a lot of work are ``astropy-healpix``,
+``uncertainties`` and the binary wheels. If any of those parts gets stuck or we
+don't have the manpower to do it, they can be done later, they aren't blockers
+for v1.0.
+
+For conda, we should strongly consider introducing a ``gammapy-all`` metapackage
+instead of the current ``gammapy-environment.yml`` shipped via ``gammapy.org``.
+That's what e.g. the Fermi tools and glueviz and others do, and they like that
+solution. This could be done any time, that change is independent of the rest of
+the changes described here.
+
+We could support other ways to ship Gammapy, e.g. Homebrew (works on MacOS and
+also Linux now), or Docker images (works anywhere).
+
+Task list
+=========
+
+Following the Gammapy v0.13 release in July 2019, the following changes should
+be made, which will ship to users in Gammapy v0.14 in September 2019:
+
+- Maybe: update Gammapy packaging and dev tools to follow new recommendations in `APE 17`_.
+  There's some advantages to doing this first, to make it easier to define the
+  new CI test matrix, but this could also come later.
+- Update continuous integration test matrix to test against the minimum required
+  versions as specified here (and also newer versions in addition) (`GH 2270`_).
+  Maybe move completely from travis-ci to Azure?
```

Comment:
Is it related?

Reply:
I think the CI update should be mentioned. It's one of the larger pieces of work actually, and is absolutely needed to ensure that what we describe here works and is supported. I removed the ? concerning travis-ci vs azure though, that's something we'll just decide when we work on this.